stripe/_quote_service.py

```diff
@@ -3053,6 +3267,27 @@ def pdf(
             ),
         )
 
+    async def pdf_async(
+        self,
+        quote: str,
+        params: "QuoteService.PdfParams" = {},
+        options: RequestOptions = {},
+    ) -> Any:
+        """
+        Download the PDF for a finalized quote
+        """
+        return cast(
+            Any,
+            await self._request_stream_async(
```

Comment:
Looks like `_request_stream_async` is missing in `StripeService`

Reply:
Fixed in [Stream](https://github.com/stripe/stripe-python/pull/1228/commits/24023f32f48db10dbdaf635a4624a2035552d43c)